<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindCmpWin</class>
 <widget class="QMainWindow" name="FindCmpWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>279</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find text window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RealCompare.qrc">
    <normaloff>:/Resources/img/main.png</normaloff>:/Resources/img/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="findinfilesTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_find">
       <attribute name="title">
        <string>find</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Find what :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="findComboBox">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="maxCount">
               <number>512</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="findBackwardBox">
               <property name="text">
                <string>Backward direction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="findMatchWholeBox">
               <property name="text">
                <string>Match whole word only</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="findMatchCaseBox">
               <property name="text">
                <string>Match case</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="findWrapBox">
               <property name="text">
                <string>Wrap around</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Search Mode</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QRadioButton" name="findModeRegularBt">
                 <property name="text">
                  <string>Regular expression</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="findModeNormalBt">
               <property name="text">
                <string>Normal</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="findTextNext">
            <property name="text">
             <string>Find Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="findCloseBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Diretion</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="leftSearch">
               <property name="text">
                <string>Search In Left</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rightSearch">
               <property name="text">
                <string>Search In Right</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="RealCompare.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>findTextNext</sender>
   <signal>clicked()</signal>
   <receiver>FindCmpWin</receiver>
   <slot>slot_findNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findCloseBt</sender>
   <signal>clicked()</signal>
   <receiver>FindCmpWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_findNext()</slot>
  <slot>slot_findCount()</slot>
  <slot>slot_findAllInCurDoc()</slot>
  <slot>slot_findAllInOpenDoc()</slot>
  <slot>slot_replace()</slot>
  <slot>slot_replaceFindNext()</slot>
  <slot>slot_replaceAll()</slot>
  <slot>slot_replaceAllInOpenDoc()</slot>
 </slots>
</ui>
